Source/Core/Core/PrimeHack/Mods/FpsControls.cpp

```diff
@@ -1060,6 +1081,8 @@ namespace prime {
       mp1_static.lockon_address = 0x804c00b3;
       mp1_static.tweak_player_address = 0x804ddff8;
       mp1_static.cplayer_address = 0x804d3c20;
+      mp1_static.object_list_ptr_address = 0x804bfc30;
+      mp1_static.camera_uid_address = 0x804c4a08;
```

Comment:
I don't think these addresses are static.

Reply:
I just stole them from the noclip code, quite literally copy pasted their name and everything. 

If that's the case then there might be some fixing in the noclip code to do as well

Reply:
Our rule of thumb is generally anything around `804bxxxx` or lower is static, we've had no reports or issues so far with these addresses so far.